src/service/video-manager-impl.js

```diff
@@ -376,10 +376,14 @@ class VideoEntry {
    * @private
    */
   videoPaused_() {
-    const trackingVideo = assertTrackingVideo(this.video);
-    if (trackingVideo &&
-        trackingVideo.getCurrentTime() !== trackingVideo.getDuration()) {
-      this.analyticsEvent_(VideoAnalyticsType.PAUSE);
+    const v = this.video;
+    if (isTrackingVideo(v)) {
+      const trackingVideo =
```

Comment:
Why is the type cast needed? How are `#getCurrentTime` and `#getDuration` not implemented for a normal video?